src/wp-includes/rest-api/fields/class-wp-rest-meta-fields.php

```diff
@@ -79,10 +79,9 @@ public function get_value( $object_id, $request ) {
 		foreach ( $fields as $meta_key => $args ) {
 			$name       = $args['name'];
 			$all_values = get_metadata( $this->get_meta_type(), $object_id, $meta_key, false );
-
 			if ( $args['single'] ) {
 				if ( empty( $all_values ) ) {
-					$value = $args['schema']['default'];
```

Comment:
Why do we need to drop this?

Reply:
All defaults are now handled upstream in the meta functions. This not required and doesn't do anything. It also currently generates a notice error.

Reply:
How so? If someone is setting the default with the schema it seems like this won't work anymore?

What notice is currently generated? I'm not seeing that on trunk.

Reply:
I pushed a failing test showing this `test_set_default_in_schema`.

Reply:
Restoring that check works for me and I'm not seeing any notices. What test data caused the warning?